crates/ty_python_semantic/resources/mdtest/class/super.md

```diff
@@ -279,6 +278,8 @@ class A[T]:
 
 class B[T](A[T]):
     def f(self, b: T) -> T:
+        # TODO: After resolving https://github.com/astral-sh/ruff/issues/17432 this should be resolved
```

Comment:
https://github.com/astral-sh/ruff/issues/17432 is resolved, is there something else causing this?

You might need to include the identity specialization of the class's generic context when creating the annotation for `self`

Reply:
sorry I linked an unrelated link. I could reproduce this issue on main with other types. I think in generic classes if `self` has a type then `super` call emits a diagnotic. I created an issue:
https://github.com/astral-sh/ty/issues/697